tensorflow/buildspec-graviton.yml

```diff
@@ -33,6 +37,20 @@
       tag_python_version: &TAG_PYTHON_VERSION py38
       os_version: &OS_VERSION ubuntu20.04
       tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2" ]
-      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton.ec2., *DEVICE_TYPE ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton., *DEVICE_TYPE ]
       context:
         <<: *INFERENCE_CONTEXT
+    BuildSageMakerTensorflowCPUInferencePy3DockerImage:
+      <<: *INFERENCE_REPOSITORY
+      build: &TENSORFLOW_CPU_INFERENCE_PY3 false
+      image_size_baseline: 4899
+      device_type: &DEVICE_TYPE cpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py38
+      os_version: &OS_VERSION ubuntu20.04
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker" ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.graviton., *DEVICE_TYPE ]
```

Comment:
Did we skip ec2 here because this is Sagemaker image?

Reply:
the EC2 is above. This is a sagemaker image. there are 2 images in this file.